src/main/java/com/icthh/xm/ms/otp/repository/UaaRepository.java

```diff
@@ -0,0 +1,77 @@
+package com.icthh.xm.ms.otp.repository;
+
+import com.icthh.xm.commons.tenant.TenantContextUtils;
+import com.icthh.xm.commons.tenant.TenantKey;
+import com.icthh.xm.commons.tenant.internal.DefaultTenantContextHolder;
+import io.github.jhipster.config.JHipsterProperties;
+import lombok.RequiredArgsConstructor;
+import org.apache.commons.lang.StringUtils;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.core.ParameterizedTypeReference;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Component;
+import org.springframework.util.LinkedMultiValueMap;
+import org.springframework.util.MultiValueMap;
+import org.springframework.web.client.RestTemplate;
+
+import javax.annotation.PostConstruct;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import static com.icthh.xm.ms.otp.config.Constants.ACCESS_TOKEN;
```

Comment:
please reorder imports to codestyle

Reply:
what code style ?

Reply:
@ishkurko  please send link to code style